File: tournaments/arrayMinimumIndex/arrayMinimumIndex.js
function arrayMinimumIndex(inputArray) {
    return inputArray.indexOf(Math.min(...inputArray));
}

function arrayMinimumIndex(inputArray) {

    var indexOfMinimum = 0;
    for (var i = 1; i < inputArray.length; i++) {
        if (inputArray[i] < inputArray[indexOfMinimum]) {
            indexOfMinimum = i;
        }
    }
    return indexOfMinimum;
}
    

File: tournaments/arrayMinimumIndex/arrayMinimumIndex.pl
#!/usr/bin/perl
use strict;
use warnings FATAL => 'all';

sub arrayMinimumIndex {
    my ($inputArray) = @_;

    use List::Util qw( min first );

    return first { $$inputArray[$_] == min(@$inputArray) } 0..@$inputArray;
}
